packages/core/test/toPromise.test.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  ActorRef,
+  createActor,
+  createMachine,
+  fromPromise,
+  toPromise
+} from '../src';
+
+describe('toPromise', () => {
+  it('should be awaitable', async () => {
+    const promiseActor = createActor(
+      fromPromise(() => Promise.resolve(42))
+    ).start();
+
+    const result = await toPromise(promiseActor);
+
+    ((_accept: number) => {})(result);
```

Comment:
@davidkpiano what is this for?

Reply:
Just asserting the type. We can do it using `satisfies`